.travis.yml

```diff
@@ -106,7 +106,7 @@ script:
     # Test packages pararell (xargs -P)
     # Also build the docker container (with embedded build info), that will be pushed to dockerhub later on.
     - if [ "$JOB_TYPE" = compile_and_basic_tests ]; then
-        go list ./... | grep -v vendor | xargs -n1 -I {} -P 4 go test -v -covermode=atomic -coverprofile=../../../{}/coverage.txt {} ;
+        go list ./... | grep -v vendor | xargs -n1 -I {} -P 4 go test -v -covermode=atomic -coverprofile=../../../{}/coverage.txt {} || exit $? ;
```

Comment:
My god, are you kidding me!? Travis shell handling is really broken. But the value 1 should be good enough for our uses.